lib/timeline/component/Group.js

```diff
@@ -475,6 +493,15 @@ Group.prototype.order = function() {
 Group.prototype._updateVisibleItems = function(orderedItems, oldVisibleItems, range) {
   var visibleItems = [];
   var visibleItemsLookup = {}; // we keep this to quickly look up if an item already exists in the list without using indexOf on visibleItems
+   
+  if (!this.isVisible) {
```

Comment:
This is the problem because examples with no groups are no longer working. Please rethink this lines again.